react-backend/client/src/components/admin/adminVisitsPage.js

```diff
@@ -67,6 +95,15 @@ export class AdminVisitsPage extends Component {
     return this.state.reauthenticated ? (
       <div>
         <h1>Visitor Data</h1>
+        <select onChange={this.filter}>
+          <option defaultValue value="">
```

Comment:
This default value of an empty string breaks the query if you switch back to it. It should default to the standard sort (date I guess?)